addon/templates/components/bread-crumb.hbs

```diff
@@ -7,9 +7,11 @@
     {{/if}}
   {{/link-to}}
 {{else}}
-  {{#if hasBlock}}
-    {{yield this route}}
-  {{else}}
-    {{route.title}}
-  {{/if}}
+  <span>
```

Comment:
I don't think the `<span>` should wrap the `yield` as it would prevent you from using any other elements.
Maybe consider it as a default for the case where the is no block given?